src/core_plugins/kibana/public/visualize/editor/editor.js

```diff
@@ -293,14 +294,21 @@ uiModules
 
   function transferVisState(fromVis, toVis, stage) {
     return function () {
+
```

Comment:
this function is invoked with the angular-binding  in `sidebar.html` (cf. `stageEditableVis`)